example/rds-cluster-tf/main.tf

```diff
@@ -27,4 +27,5 @@ module "rds_cluster" {
   snapshot_identifier = var.snapshot_identifier
   lambda_timeouts     = var.lambda_timeouts
   lambda_memory_sizes = var.lambda_memory_sizes
+  enable_upgrade      = var.enable_upgrade
```

Comment:
(making notes as I go through the upgrade)

This isn't a change from this PR but it looks like `parameter_group_family` should be here in our example configuration?

Reply:
@npauzenga is there a reason we need it generally?        I suppose you could argue testing, generally I've just modified source as being just as straightforward when that's needed.

Reply:
Yeah so I just encountered it trying to test. I think it would make sense to pass in all possible variables in the example deploy? If it's not there it's not a huge deal so not willing to die on this hill 😄

Reply:
No, that's fine and reasonable feedback: 

34b0b123e6